### ISSUE TITLE
High Contrast Mode

### ISSUE THREAD

**Issue by alisony755** [alisony755/alison-ye-website#4]
Accessibility Menu
- Make background for buttons black and button text white
- Make title (Accessibility Options) black
- Darken buttons once clicked

General
- Make page actually high contrast (white blocks instead of dark gray)
- Rename the current 'high-contrast mode' to 'dark mode'
